gooey/python_bindings/gooey_parser.py

```diff
@@ -68,6 +68,11 @@ def __init__(self, **kwargs):
         self.__dict__['parser'] = ArgumentParser(**kwargs)
         self.widgets = {}
         self.options = {}
+        if 'parents' in kwargs:

```

Comment:
Your example from the PR comments would be a fine test case for this. 

```
def test_parser_parents(): 
    base_parser = GooeyParser(add_help=False)
    base_parser.add_argument("a_file", widget="FileChooser")
    parser = GooeyParser(parents=[base_parser])
    parser.add_argument("b_file", widget="FileChooser")
  
    # assert that widgets and options have been merged here
```

I'd also like to see a test case where it does the 'wrong' thing and overwrites the parent options, which was def unexpected to me that it'd be valid (see my main comment).